jobs/ci-kubernetes-e2e-gci-gke-ingress.env

```diff
@@ -1,2 +1 @@
 ### job-env
-KUBE_NODE_OS_DISTRIBUTION=gci
```

Comment:
I don't think this can be deleted until #3725 goes in, so I'm not sure this test is operating correctly. `KUBE_GKE_IMAGE_TYPE` doesn't drive `KUBE_NODE_OS_DISTRIBUTION`, and deployment=gke => `KUBE_GKE_IMAGE_TYPE`.